configs/tsconfig-compass/tsconfig.common.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "@mongodb-js/tsconfig-devtools/tsconfig.common.js"
+  "extends": "@mongodb-js/tsconfig-devtools/tsconfig.common.json",
+  "compilerOptions": {
+    "moduleResolution": "Node16"
+  }
```

Comment:
Not moving it to shared for now just to check and make sure this doesn't break anything in other monorepos

Reply:
Yep, had to revert this